app/scripts/controllers/deploymentConfig.js

```diff
@@ -278,7 +279,7 @@ angular.module('openshiftConsole')
         };
 
         $scope.startLatestDeployment = function() {
-          if ($scope.canDeploy()) {
+          if ($scope.canDeploy() && !$scope.deploymentConfig.spec.paused) {
```

Comment:
why not include that as part of canDeploy?  is canDeploy only doing the auth check today, i thought it did more than that